plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/conf/hydra/launcher/submitit.yaml

```diff
@@ -0,0 +1,35 @@
+# @package _group_
```

Comment:
This is a good opportunity to port it to use Structured Configs in a followup diff.

Reply:
Create https://github.com/facebookresearch/hydra/issues/623 for this.